api/src/main/java/io/opentelemetry/metrics/DoubleValueRecorder.java

```diff
@@ -16,39 +16,54 @@
 
 package io.opentelemetry.metrics;
 
-import io.opentelemetry.metrics.DoubleMeasure.BoundDoubleMeasure;
+import io.opentelemetry.metrics.DoubleValueRecorder.BoundDoubleValueRecorder;
 import java.util.Map;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * Measure to report instantaneous measurement of a double value.
+ * ValueRecorder is a non-additive synchronous instrument useful for recording any non-additive
```

Comment:
The use of "non-additive" here is confusing. This means non-pre-summed, correct? I think I might just leave out the "non-additive" term here, since I don't think people will understand what is meant.

Reply:
Done.